### ISSUE TITLE
View page information even if it passes WCAG

### ISSUE THREAD

**Issue by douga11y** [ThePacielloGroup/ARC-Toolkit#4]
**Is your feature request related to a problem? Please describe.**
> The previous version of ARC Toolkit allowed you to view information like headings, lists, tables, tab order, etc. on the page regardless of whether or not it was flagged as a failure. This was very useful while manually testing pages as we could check to make sure the structure made sense within the context of the content (i.e., past what automated tools could detect).

**Describe the solution you'd like**
> Add these features into the rebuilt version of Toolkit.

**Describe alternatives you've considered**
> Alternate solutions would require using other tools like WAVE to visualize the same information.

**Additional context**
> Not looking for anything above and beyond what the previous version had. Just looking for parody. I've interacted with clients who rely on this tool to provide exploratory information about elements on the page. It's important that we consider this tool is not only being used by people in ARC Capture for reporting failures!

**Comment by poorgeek** [comment]
With the way that the ARC Rules engine now returns results, this is a little more complex to address than in previous versions, particularly with regards to keeping Toolkit performant when potentially display large amounts of data. I've rescheduled this for later this year to allow for more time to work on the UX and performance around this feature.

**Comment by douga11y** [comment]
The highlight features added with 5.4 is a huge step in this direction. It allows for the visualization tools I was looking for when I opened this. Thanks for all the hard work that went into this update!
